src/main/java/com/google/api/codegen/config/FlatteningConfig.java

```diff
@@ -138,31 +149,49 @@
   @Nullable
   private static Map<String, FlatteningConfig> createFlatteningConfigsFromProtoFile(
```

Comment:
The name of this method suggests it only looks at the proto file, but now we are mixing in the gapic config. Can we rename? And also, can we clarify (either in method comments or inline comments somewhere) exactly what behaviour we expect from the different cases of gapic config being either present or missing? E.g. if we have a gapic config, should we ignore all proto annotation flattenings? If not, why not?

Reply:
There is no difference in behavior, for this method, if the GAPIC config is missing or not.

Ok, updated the javadoc comment to answer those questions. Hows' that look?

Reply:
That looks good. But I also wanted to clarify that the list of flattenings generated will include flattenings from *both* the gapic config and the proto annotations. Is that right? And is that what we want to happen?

Reply:
ok this is the last straw, i'll draft up a proposal for handling both config specs :)